tools/PROTOS.sh

```diff
@@ -51,7 +51,7 @@ function clone_common_protos {
 # Require a specific version of protoc for generating files.
 # This stabilizes the generated file output, which includes the protoc version.
 . tools/PROTOC-VERSION.sh
-if [ "$(protoc --version)" != "libprotoc $PROTOC_VERSION" ]; then
-    echo "Please update your protoc to version $PROTOC_VERSION, the current required version as specified in tools/PROTOC_VERSION.sh"
+if [ "$(protoc --version)" != "libprotoc 3.$PROTOC_VERSION" ]; then
+    echo "Please update your protoc to version 3.$PROTOC_VERSION, the current required version as specified in tools/PROTOC_VERSION.sh"
```

Comment:
So now it seems that "3" is the $GO_PROTOC_MAJOR_VERSION. Someday we might want to make that explicit instead of leaving the mysterious "3." around, which someone in the future might think is a typo.